charts/calhoun/values.yaml

```diff
@@ -0,0 +1,168 @@
+
+probes:
+  readiness:
+    # probes.readiness.enabled -- Whether to configure a readiness probe
+    enabled: true
+    spec:
+      httpGet:
+        path: /status
+        port: 8080
+      timeoutSeconds: 5
+      initialDelaySeconds: 20
+      periodSeconds: 10
+      failureThreshold: 6
+      successThreshold: 1
+  liveness:
+    # probes.liveness.enabled -- Whether to configure a liveness probe
+    enabled: true
+    spec:
+      httpGet:
+        path: /status
+        port: 8080
+      timeoutSeconds: 5
+      initialDelaySeconds: 15
+      periodSeconds: 10
+      failureThreshold: 30 # 5 minutes before restarted
+      successThreshold: 1
+  startup:
+    # probes.startup.enabled -- Whether to configure a startup probe
+    enabled: true
+    spec:
+      httpGet:
+        path: /status
+        port: 8080
+      timeoutSeconds: 5
+      periodSeconds: 10
+      failureThreshold: 6
+      successThreshold: 1
+
+vault:
+  # vault.enabled -- When enabled, syncs required secrets from Vault
+  enabled: true
+  # vault.pathPrefix -- (string) Vault path prefix for secrets. Required if vault.enabled.
+  pathPrefix:
+
+ingress:
+  # ingress.enabled -- Whether to create Ingress, Service and associated config resources
+  enabled: true
+  # ingress.staticIpName -- (string) Required. Name of the static IP, allocated in GCP, to associate with the Ingress
+  staticIpName: null
+  # ingress.sslPolicy -- (string) Name of a GCP SSL policy to associate with the Ingress
+  sslPolicy: null
+  # ingress.securityPolicy -- (string) Name of a GCP Cloud Armor security policy
+  securityPolicy: null
+  # ingressTimeout -- (number) number of seconds requests on the https loadbalancer will time out after
+  timeoutSec: 120
+
+  domain:
+    # domain.hostname -- Hostname of this deployment
+    hostname: calhoun
+    # domain.namespaceEnv -- If true, an extra level of namespacing (`global.terraEnv`) will be added between the hostname and suffix
+    namespaceEnv: true
+    # domain.suffix -- Domain suffix
+    suffix: dsde-dev.broadinstitute.org
+
+  cert:
+    # A TLS certificate is used for the GKE ingress and the OIDC proxy sidecar.
+    # If preSharedCerts is not empty, its contents are used for the GKE ingress. Otherwise a secret with the cert is expected.
+    # The cert secret is required if the OIDC proxy is enabled.
+    # This secret can be
+    #   - synced from Vault
+    #   - created/managed with cert-manager
+    #   - created manually before the chart is applied
+
+    # ingress.cert.preSharedCerts -- Array of pre-shared GCP SSL certificate names to associate with the Ingress
+    preSharedCerts: []
+
+    # If cert.vault is enabled, a secrets-manager SecretDefinition resource is created to sync the
+    #   certificate from Vault to a k8s secret.
+    vault:
+      # ingress.cert.vault.enabled -- Enable syncing certificate secret from Vault. Requires [secrets-manager](https://github.com/tuenti/secrets-manager)
+      enabled: true
```

Comment:
Can we have this default to false? We actually use the `preSharedCerts` value as the main method of setting up tls in the persistent environments (dev, perf, alpha, ...etc)